src/ert/config/queue_config.py

```diff
@@ -20,11 +19,53 @@
 
 GENERIC_QUEUE_OPTIONS: List[str] = ["MAX_RUNNING"]
 VALID_QUEUE_OPTIONS: Dict[Any, List[str]] = {
-    QueueSystem.TORQUE: _clib.torque_driver.TORQUE_DRIVER_OPTIONS
-    + GENERIC_QUEUE_OPTIONS,
-    QueueSystem.LOCAL: [] + GENERIC_QUEUE_OPTIONS,  # No specific options in driver
-    QueueSystem.SLURM: _clib.slurm_driver.SLURM_DRIVER_OPTIONS + GENERIC_QUEUE_OPTIONS,
-    QueueSystem.LSF: _clib.lsf_driver.LSF_DRIVER_OPTIONS + GENERIC_QUEUE_OPTIONS,
+    QueueSystem.TORQUE: GENERIC_QUEUE_OPTIONS
+    + [
+        "CLUSTER_LABEL",
+        "DEBUG_OUTPUT",
+        "JOB_PREFIX_KEY",
+        "KEEP_QSUB_OUTPUT",
+        "MEMORY_PER_JOB",
+        "NUM_CPUS_PER_NODE",
+        "NUM_NODES",
+        "QDEL_CMD",
+        "QSTAT_CMD",
+        "QSTAT_OPTIONS",
+        "QSUB_CMD",
+        "QUEUE",
+        "QUEUE_QUERY_TIMEOUT",
+        "SUBMIT_SLEEP",
+    ],
+    QueueSystem.LOCAL: GENERIC_QUEUE_OPTIONS,
+    QueueSystem.SLURM: GENERIC_QUEUE_OPTIONS
+    + [
+        "SBATCH",
+        "SCONTROL",
+        "SQUEUE",
+        "SCANCEL",
+        "PARTITION",
+        "SQUEUE_TIMEOUT",
+        "MAX_RUNTIME",
+        "INCLUDE_HOST",
+        "EXCLUDE_HOST",
+    ],
+    QueueSystem.LSF: GENERIC_QUEUE_OPTIONS
+    + [
+        "LSF_QUEUE",
+        "LSF_RESOURCE",
+        "LSF_SERVER",
+        "LSF_RSH_CMD",
+        "LSF_LOGIN_SHELL",
+        "BSUB_CMD",
+        "BJOBS_CMD",
+        "BKILL_CMD",
+        "BHIST_CMD",
+        "BJOBS_TIMEOUT",
+        "DEBUG_OUTPUT",
```

Comment:
I think this was deprecated? At least, it seemed not to be implemented in the code any longer. But I might have missed it.

Reply:
They are in the old code, so I have just copied it.

Reply:
These stem from the list in the c++ header file.

Reply:
(or trying to :smile: )
![image](https://github.com/equinor/ert/assets/1692372/662a5393-194f-4bba-a1bc-f790ba8ac5d2)